build.sbt

```diff
@@ -263,6 +263,11 @@ lazy val rocc_acc_utils = (project in file("generators/rocc-acc-utils"))
   .settings(libraryDependencies ++= rocketLibDeps.value)
   .settings(commonSettings)
 
+lazy val hls_accel = (project in file("generators/hls-example"))
```

Comment:
I think this should go in generators/chipyard/example, instead of its own project